code/examples/channel/src/store.rs

```diff
@@ -93,6 +97,14 @@ impl Db {
             value.and_then(|value| decode_certificate(&value.value()).ok())
         };
 
+        let values_bytes = value.unwrap().to_bytes()?.to_vec().len() as u64;
+        let certificates_bytes = encode_certificate(&certificate.clone().unwrap())?.len() as u64;
+
+        self.metrics
+            .add_read_bytes(values_bytes + certificates_bytes);
+        self.metrics
+            .add_key_read_bytes(Height::to_bytes(&height)?.len() as u64);
```

Comment:
Here we can probably use `size_of::<Height>` instead of performing the conversion.